### ISSUE TITLE
Audio not playing on Android. Yes in browser

### ISSUE THREAD

**Issue by matias-tecnosoul** [EltonFaust/cordova-plugin-multi-player#17]
Hi, I've been using the plugin for a couple of years flawlessly, but now I'm having an issue with some apps, not all :P
The player works fine on the browser but when I run it on Android it won´t play.
I tried removing the platform and plugins and readding them but still wont work.

When I debug with adb I can see this message when playing the stream:
```

06-24 18:01:27.407  9157  9460 V MultiPlayer: Plugin : RADIO STATE - LOADING...
06-24 18:01:27.417  9157  9460 V MultiPlayer: RadioPlayerService : Player state changed. ExoPlayer State: 2, Current state: STOPPED
06-24 18:01:27.417  9157  9460 V MultiPlayer: RadioPlayerService : Player state changed. ExoPlayer State: 2, Current state: STOPPED
06-24 18:01:30.465  9157  9460 V MultiPlayer: Plugin : RADIO STATE - ERROR...
06-24 18:01:30.466  9157  9460 V MultiPlayer: RadioPlayerService : ERROR OCCURED.
06-24 18:01:30.466  9157  9460 V MultiPlayer: RadioPlayerService : Player state changed. ExoPlayer State: 1, Current state: STOPPED
```
In other of my apps I can see in adb:

```
06-24 18:03:12.705 12338 12495 V MultiPlayer: Plugin : ACTION - play
06-24 18:03:12.705 12338 12495 V MultiPlayer: Plugin : RADIO STATE - LOADING...
06-24 18:03:12.715 12338 12495 V MultiPlayer: RadioPlayerService : Player state changed. ExoPlayer State: 2, Current state: STOPPED
06-24 18:03:12.715 12338 12495 V MultiPlayer: RadioPlayerService : Player state changed. ExoPlayer State: 2, Current state: STOPPED
06-24 18:03:14.496 12338 12495 V MultiPlayer: RadioPlayerService : Player state changed. Playing
06-24 18:03:14.497 12338 12495 V MultiPlayer: Plugin : RADIO STATE - PLAYING...
```

I'm on
cordova: 11.0.0
cordova-plugin-androidx-adapter 1.1.3 "cordova-plugin-androidx-adapter"
cordova-plugin-androidx 3.0.0 "cordova-plugin-androidx"
cordova-plugin-multi-player 2.1.3 "Multi Player Plugin"
cordova-plugin-music-controls2 3.0.6 "Music Controls"

Installed platforms:
  android 10.1.2
  browser 6.0.0

As I mention before, I'm not having the issue with other apps using similar / identical settings.
Any hits please!?
thanks in advance!

**Comment by matias-tecnosoul** [comment]
can I see more details of the error?
thanx!

**Comment by matias-tecnosoul** [comment]
I could finally trace the error down to an encryption issue... 
Somehow the ssl conection to the Icecast source was giving an error (that I couldnt find out yet). So I was trying with a plain http conection but it was still giving me problems... I finally found I needed to append a "usesCleartextTraffic true " to Android Manifest...
Its working for the time being. Ill try to dig the ssl error later on...
thanks!
